src/scss/dp/overrides/helpers/_utilities.scss

```
@@ -665,3 +665,10 @@ $old-ie: false;
 .clearfix {
     @extend %clearfix;
 }
+
+// Highlighting - temporary fix while waiting for the design system to include highlighting in their npm package
+.ons-highlight {
+  background-color: $color-highlight;
+  font-style: normal;
+  padding: 0 2px;
+}
```

There's a new [file](src/scss/dp/overrides/design-system/_utilities.scss) where these utility classes should live. This should be where Sixteens utilities live and will be deleted when we migrate over.

The naming convention should be `ons-u...`
```suggestion

// Highlighting - temporary fix while waiting for the design system to include highlighting in their npm package
.ons-u-highlight {
  background-color: $color-highlight;
  font-style: normal;
  padding: 0 2px;
}
```

---

Changing the naming convention would break highlighting once the npm package is updated in the design system. We're hoping to avoid that situation.

---

I've found the [slack conversation](https://onswebsite.slack.com/archives/C01D1D9JM33/p1648204707343599) 😓. As it's not a utility class and as I mentioned above ⬆️ the file is not the best location for design-system overrides. I think the override should live in a similar place ([../overrides/design-system/_foundations.scss](https://github.com/ONSdigital/dp-design-system/blob/main/src/scss/dp/overrides/design-system/_foundations.scss)) to the [design system](https://github.com/ONSdigital/design-system/tree/master/src/foundations/style/typography/highlighting) but ignore my suggestion of adding the `u`